packages/main/test/specs/DatePicker.spec.js

```diff
@@ -223,7 +223,7 @@ describe("Date Picker Tests", () => {
 		const valueHelpIcon = await datepicker.getValueHelpIcon();
 		await valueHelpIcon.click();
 
-		let calendarDate_4_Jan_2019 = await datepicker.getPickerDate(1546560000); //Jan 4, 2019
+		let calendarDate_4_Jan_2019 = await datepicker.getPickerDate(1638576000); //Jan 4, 2019
 		await calendarDate_4_Jan_2019.click();
 
 		const innerInput = await datepicker.getInnerInput();
```

Comment:
I do not understand that part, the date is different now, but the input stays the same.
If the date is different, please change the comments and variable names also.

Reply:
I've disabled the failing test as it works based on a faked Date.prototype.getDate method.
That way a DateFormat instance with strict parsing enabled is unable to parse the value strings from the input field.
I would suggest to address this issue in another change.

Reply:
agree

Reply:
@unazko In case you don't plan to immediately fix it with another change. please create an issue for that not to forget it. If you plan to immediately address it, a new issue is not required.

Reply:
Additional investigation time is needed, in order to adjust the described test properly.
I've created the following issue in this regard: https://github.com/SAP/ui5-webcomponents/issues/4443

Reply:
Thanks